app/assets/javascripts/discourse/components/reviewable-item.js.es6

```diff
@@ -0,0 +1,162 @@
+import { ajax } from "discourse/lib/ajax";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+import computed from "ember-addons/ember-computed-decorators";
+import Category from "discourse/models/category";
+import optionalService from "discourse/lib/optional-service";
+
+let _components = {};
+
+export default Ember.Component.extend({
+  adminTools: optionalService(),
+  tagName: "",
+  updating: null,
+  editing: false,
+  _updates: null,
+
+  @computed("reviewable.type")
+  customClass(type) {
+    return type.dasherize();
+  },
+
+  @computed("reviewable.score")
+  displayScore(score) {
+    return score.toFixed(1);
+  },
+
+  // Find a component to render, if one exists. For example:
+  // `ReviewableUser` will return `reviewable-user`
+  @computed("reviewable.type")
+  reviewableComponent(type) {
+    if (_components[type] !== undefined) {
+      return _components[type];
+    }
+
+    let dasherized = Ember.String.dasherize(type);
```

Comment:
```suggestion
    const dasherized = Ember.String.dasherize(type);
```

Reply:
I don't feel as strongly about making everything `const` as I used to. I find it doesn't catch many bugs, and nothing seems faster as a result (which is initially what I'd hoped.)

Reply:
No it mostly doesn't do anything, I agree. In the end all is transpiled into `var`. The only thing it does is hitting the developer about the fact that this is not going to be mutated later, which I do appreciate, but not strong feelings about this.

Reply:
My current feeling on it is it's up to the developer writing the code. Using `let` or `const` fixes the weird Javascript scope issues, and they should choose what they think is more appropriate. I'll review/approve either form personally.

Reply:
works for me :+1: